zstacklib/zstacklib/test/test_singleflight.py

```
@@ -78,6 +78,33 @@ def worker():
+            import random
+            if random.random() < 0.2:
+                raise Exception("Sensor read failed")
+            return "Temperature: 25°C"
+
+        try:
+            result = self.sft.do(key, fetch)
+            if result.error:
+                print("Error: {result.error}")
+                return "Fallback data"
+            return result.value
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)